firebase.json

```diff
@@ -3,50 +3,45 @@
     "rules": "firestore.rules",
     "indexes": "firestore.indexes.json"
   },
+  "storage": {
+    "rules": "firestore.rules"
```

Comment:
If you can have a separate file, should we? Seems cleaner.

Reply:
Actually looks like this should be `storage.rules`

Reply:
woops was a typo, i meant storage.rules